### PR TITLE
Add Teddy Doge V2 (BEP20)

### PR DESCRIPTION
**Pull request by DRAC-Network**
Adding info for BEP20 token 'Teddy Doge V2'.

PR created by <a href="https://assets.trustwallet.com">TW Assets Management</a>

### CODE REVIEW

**Comment by Katelove11** [comment]
Hi

**Comment by cB1980623** [comment]
TM4gVAY6N6co159puARpgsQ6HJVQZiDQyy

**Comment by cryptomanz1** [comment]
• What is the purpose of this token? How will it benefit Trust Wallet users?
 • How is the token being distributed? Are you conducting an airdrop?
 • Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?
 • Is the token listed on a centralized/decentralized exchange?
 • Is a Smart Contract audit completed? Where can we find this information?

**Comment by DRAC-Network** [comment]
**• What is the purpose of this token? How will it benefit Trust Wallet users?**

TeddyDoge is A one-stop full-featured decentralized exchange.
Including a swap, aggregate transactions, liquidity, farm, 15 mainstream chain transactions, 15 mainstream cross-chains, 15 mainstream chain chart system, token and liquidity multi-scheme locking, TeddyWallet and derivatives.

The TeddyDoge community currently has a growing community of 51,000 followers, which can bring more people to Trust Wallet and improve its ecosystem for the benefit of Trust Wallet users.

**• How is the token being distributed? Are you conducting an airdrop?**

Our total issuance is 1285199693195170 * 1e18 tokens and we did not conduct airdrops. Because the TeddyV2 token is the replacement token for TeddyV1, the token distribution is only 20% liquidity addition and 80% customer replacement.

**• Did you sold the tokens at an Initial Coin Offering (ICO) or a token presale?**

There is no ICO or pre-sale for our tokens, because our TeddyV2 tokens are an alternative token for users to exchange TeddyV1 tokens, so only users use TeddyV1 through the swap page
Token swap.

**• Is the token listed on a centralized/decentralized exchange?**

Yes,our tokens have been listed on pancakeswap, token swap link:
https://pancakeswap.finance/swap?outputCurrency=0xDB79c12d1d0670988A39B0E48b96e955eF922d24

**• Is a Smart Contract audit completed? Where can we find this information?**

Our token contract has passed Bensin's security audit, the link to the security audit report:
https://beosin.com/audits/Teddy%20V2_202208121455.pdf

File: blockchains/smartchain/assets/0xDB79c12d1d0670988A39B0E48b96e955eF922d24/info.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{
+    "name": "Teddy Doge V2",
+    "type": "BEP20",
+    "symbol": "Teddy V2",
+    "decimals": 18,
+    "website": "A one-stop full-featured decentralized exchange.",
```
Comment (cryptomanz1) [inline blockchains/smartchain/assets/0xDB79c12d1d0670988A39B0E48b96e955eF922d24/info.json:6]: You need add link for website.

**Comment by cryptomanz1** [comment]
Your logo should be a full screen, please change.

**Comment by DRAC-Network** [comment]
> 您的徽标应该是全屏，请更改。
After confirmation, the team decided to use the original submitted logo without modification. Hope you can approve.
Thanks!

**Comment by Cryptocool1** [comment]
Your logo should be full screen size.

**Comment by Cryptocool1** [comment]
And your logo looks blurry, please change to the good quality

**Comment by DRAC-Network** [comment]
> Your logo should be full screen size.

We understand, but our logo was approved at the time of TeddyDoge V1 using the images submitted now, so the TeddyDogeV2 team did not expect any differences between them. Hope you can understand.

**Comment by DRAC-Network** [comment]
> And your logo looks blurry, please change to the good quality

Ok, got it, we're going to revise.

**Pull request closed without merging**